docs/proposed/psa-driver-wrappers-codegen-migration-guide.md

```diff
@@ -2,6 +2,7 @@ Migrating to an auto generated psa_crypto_driver_wrappers.c file
 ================================================================
 
 **This is a specification of work in progress. The implementation is not yet merged into Mbed TLS.**
+For a description of the current state of drivers Mbed TLS, see our [PSA Cryptoprocessor driver development examples](../psa-driver-example-and-guide.html).
```

Comment:
This document describes the generator script as it is now, so the line about a work in progress is not correct. When the generator script is completed, this document should no longer be needed: the integration guide should suffice.